### ISSUE TITLE
document how to unset snap config options

### ISSUE THREAD

**Issue by wwwtyro** [juju-solutions/bundle-canonical-kubernetes#313]
`snap set kube-apiserver whatever=null`

**Comment by tvansteenburgh** [comment]
Blocked on https://github.com/kubernetes/kubernetes/pull/47912

**Comment by Cynerva** [comment]
> Blocked on kubernetes/kubernetes#47912

I thought this issue was more documenting how to unset config with `snap set` stuff, which wouldn't be blocked by the api-extra-args feature. I guess we need to clarify what we're trying to communicate here and where it belongs.

**Comment by tvansteenburgh** [comment]
@Cynerva Yeah, my bad, I meant it's blocked on the same thing that PR is blocked on - namely the bug that breaks snapd when you unset. I don't think we should document it when we know it's broken.

**Comment by tvansteenburgh** [comment]
This is unblocked now - snapd 2.27 was released.

**Comment by tvansteenburgh** [comment]
I was wrong, snapd 2.27 hasn't been released yet, this is still blocked.
